app/_locales/es/messages.json

```diff
@@ -1,10 +1,840 @@
+  },
+  "readMore": {
+    "message": "Leer mas aqui."
+  },
+  "receive": {
+    "message": "Recibir"
+  },
+  "recipientAddress": {
+    "message": "Dirección del receptor"
+  },
+  "refundAddress": {
+    "message": "Su dirección de reembolso"
+  },
+  "rejected": {
+    "message": "Rechazado"
+  },
+  "required": {
+    "message": "Requerido"
+  },
+  "retryWithMoreGas": {
+    "message": "Vuelva a intentar con un precio de Gas más alto aquí"
+  },
+  "revert": {
+    "message": "Revertir"
+  },
+  "rinkeby": {
+    "message": "Red privada Rinkeby"
+  },
+  "ropsten": {
+    "message": "Red privada Ropsten"
+  },
+  "sampleAccountName": {
+    "message": "E.g. Mi nueva cuenta",
+    "description": "Help user understand concept of adding a human-readable name to their account"
+  },
+  "save": {
+    "message": "Guardar"
+  },
+  "saveAsFile": {
+    "message": "Guardar como archivo",
+    "description": "Account export process"
+  },
+  "selectService": {
+    "message": "Seleccionar servicio"
+  },
+  "send": {
+    "message": "Enviar"
+  },
+  "sendTokens": {
+    "message": "Enviar Tokens"
+  },
+  "sendETH": {
+    "message": "Enviar ETH"
+  },
+  "sendTokensAnywhere": {
+    "message": "Enviar Tokens a cualquiera con una cuenta de Ethereum"
+  },
+  "settings": {
+    "message": "Configuracion"
+  },
+  "info": {
+    "message": "Informacion"
+  },
+  "shapeshiftBuy": {
+    "message": "Comprar con Shapeshift"
+  },
+  "showPrivateKeys": {
+    "message": "Mostrar llaves privadas"
+  },
+  "showQRCode": {
+    "message": "Mostrar codigo QR"
+  },
+  "sign": {
+    "message": "Firmar"
+  },
+  "signMessage": {
+    "message": "Firmar Mensaje"
+  },
+  "signNotice": {
+    "message": "Firmar este mensaje puede tener\n efectos secundarios peligrosos. Solo firmar \nmensajes desde sitios en los cuales tu les confies\n tu cuenta completa. Este metodo peligroso va a ser \nremovido en una version futura."
+  },
+  "sigRequest": {
+    "message": "Solicitud de firma"
+  },
+  "sigRequested": {
+    "message": "Firma solicitada"
+  },
+  "status": {
+    "message": "Estado"
+  },
+  "submit": {
+    "message": "Enviar"
+  },
+  "takesTooLong": {
+    "message": "Taking too long?"
+  },
+  "testFaucet": {
+    "message": "Testear Faucet"
+  },
+  "to": {
+    "message": "Para:"
+  },
+  "toETHviaShapeShift": {
+    "message": "$1 a ETH via ShapeShift",
+    "description": "system will fill in deposit type in start of message"
+  },
+  "tokenBalance": {
+    "message": "Tu balance de Tokens es:"
+  },
+  "total": {
+    "message": "Total"
+  },
+  "transactionMemo": {
+    "message": "Memo de transaccion (opcional)"
+  },
+  "transactionNumber": {
+    "message": "Numero de transaccion"
+  },
+  "transfers": {
+    "message": "Transferencias"
+  },
+  "troubleTokenBalances": {
+    "message": "Tuvimos problemas para cargar sus balances de tokens. Puedes verlos",
```

Comment:
`Tuvimos problemas para cargar sus balances de tokens. Puedes verlos ` (add space at the end)